### PR TITLE
Source reorganisation

### PR DESCRIPTION
**Pull request by tintou**
It depends on the gulp branch being merged

### CODE REVIEW

**Comment by tintou** [comment]
Fixes #31

**Comment by tintou** [comment]
Re-synced with master

**Comment by tintou** [comment]
This one is probably safer to review than the lint changes

**Review by eddyparkinson — APPROVED** [review]
>Source reorganisation

Look good to me. Moves some files and folders, updates docs. Looks to impact only compile step.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ See it in action at [ethercalc.net](http://ethercalc.net)!
 * jQuery (only on the client-side)
 
 ## Usage
-You can `require('socialcalc')` or include `node_modules/socialcalc/SocialCalc.js` on your web page using a script tag.
+You can `require('socialcalc')` or include `node_modules/socialcalc/dist/SocialCalc.js` on your web page using a script tag.
```
Comment (marcelklehr) [inline README.md:11]: This will require updating EtherCalc, but looks good otherwise.

**Review by marcelklehr — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (eddyparkinson) [reply README.md:11]: True, would break this line - https://github.com/audreyt/ethercalc/blob/master/Makefile#L50

**Comment by tintou** [comment]
Pull request proposed in ethercalc https://github.com/audreyt/ethercalc/pull/582

**Comment by eddyparkinson** [comment]
**Requirement** Ethercalc version 2018-01-12    _requires these changes_

**Comment by marcelklehr** [comment]
Ah, we missed package.json: https://github.com/marcelklehr/socialcalc/blob/master/package.json#L5

**Pull request merged**